arwen/host/execution_vmoutputs_test.go

```diff
@@ -478,7 +478,7 @@ func expectedVMOutputDestCtxBuiltinDoSomething(input *vmcommon.ContractCallInput
 		nil,
 	)
 	account.Balance = big.NewInt(1000)
-	account.BalanceDelta = big.NewInt(0)
+	account.BalanceDelta = big.NewInt(0).Sub(arwen.One, arwen.One)
```

Comment:
one minus one is zero.

Reply:
This should not be needed, but `big.Int` is bugged. The following test fails:
```
func TestZeroEquality(t *testing.T) {
	zero := big.NewInt(0)
	one := big.NewInt(1)
	zERO := big.NewInt(0).Sub(one, one)
	require.Equal(t, zero, zERO)
}
```

Reply:
Cannot easily use `big.Int.Cmp()` because `account.BalanceDelta` is part of a larger `VMOutput`.